game_patch/os/commands.cpp

```diff
@@ -104,6 +107,62 @@ FunHook<void()> verify_level_cmd_hook{
     }
 };
 
+bool check_can_use_cheat_cmd()
+{
+    return (rf::level.flags & rf::LEVEL_LOADED) && !rf::is_multi;
+}
+
+void handle_camera_command(FunHook<void()>& hook)
+{
+    if (!check_can_use_cheat_cmd()) {
+        if (!(rf::level.flags & rf::LEVEL_LOADED)) {
+            rf::console::print("No level loaded!");
+        }
+        else if (rf::is_multi) {
+            rf::console::print("That command can't be used in multiplayer.");
+        }
+        return;
+    }
+    hook.call_target();
+
+    rf::CameraMode current_mode = rf::camera_get_mode(rf::local_player->cam);
```

Comment:
Can you use const for locals where able? It is considered normative and helps clarify intent. [Authoritative reference](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#con4-use-const-to-define-objects-with-values-that-do-not-change-after-construction). I like const parameters in definitions as well for consistency but it is considered "pedantic" by said guidelines.
For example:
```suggestion
    const rf::CameraMode current_mode = rf::camera_get_mode(rf::local_player->cam);
```